src/Components/Web/src/FileDownloader.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Components;
+
+/// <summary>
+/// Provides an abstraction for downloading a file from a Blazor app.
+/// </summary>
+public sealed class FileDownloader : IFileDownloader
+{
+    private IJSRuntime JS { get; set; } = default!;
```

Comment:
I _believe_ our convention for injecting services would be to have them as `field`s and prefixed with `_` in (lower) camel case.